### PR TITLE
Task #8271: Create UI for Sign Up Screen

### PR DESCRIPTION
**Pull request by tanbtd-1747**
### Issue Link :link:
https://edu-redmine.framgia.vn/issues/8271

### Goals :soccer:
- Create UI for Sign Up Screen

### CODE REVIEW

**Comment by tanbtd-1747** [comment]
ready

**Review by minhduc2511 — APPROVED** [review]

File: Money*/Money*/Scenes/Signup/SignupViewController.swift [hunk 1]
```diff
@@ -9,5 +9,27 @@
 import UIKit
 
 class SignupViewController: UIViewController {
```
Comment (hung070996) [inline Money*/Money*/Scenes/Signup/SignupViewController.swift]: add final

File: Money*/Money*/Scenes/Signup/SignupViewController.swift [hunk 2]
```diff
@@ -9,5 +9,27 @@
...
+    @IBOutlet private var cancelButton: UIButton!
+    
+    // MARK: - Private functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureSubviews()
+    }
+    
+    private func configureSubviews() {
+        emailContainerView.makeRounded()
```
Comment (hung070996) [inline Money*/Money*/Scenes/Signup/SignupViewController.swift]:
```suggestion
        [emailContainerView, passwordContainerView, confirmPasswordContainerView].forEach { $0.makeRounded() }
```

**Review by tgdong2296 — APPROVED** [review]

File: Money*/Money*/Scenes/Signup/SignupViewController.swift (hunk 2 above)
Comment (tanbtd-1747) [reply Money*/Money*/Scenes/Signup/SignupViewController.swift]: fixed

**Review by hung070996 — APPROVED** [review]

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**